napari/layers/surface/surface.py

```diff
@@ -238,8 +240,11 @@ def __init__(
         # Shading mode
         self._shading = shading
 
-        self.wireframe = wireframe or SurfaceWireframe()
-        self.normals = normals or SurfaceNormals()
+        self._wireframe = SurfaceWireframe()
+        self._normals = SurfaceNormals()
+
+        self.wireframe = wireframe
+        self.normals = normals
```

Comment:
why is this change required?

Reply:
Cause if `None` is passed to the setter, I'm running `self._normals.reset()`, so we need to have a model initialized.

Reply:
My bad. I do not notice `_` at the begin of the variable name.